### ISSUE TITLE
"A Java Exception has occurred."

### ISSUE THREAD

**Issue by vorou** [tobyweston/radiate#33]
I've tried running both .jar and .exe, both results into the exception.

Windows 8.1 x64
java version "1.7.0_60"
radiate-1.0-20140618.062538-15.exe

**Comment by Andrea** [comment]
Having the same issue here and can't find a .radiate file (tried with the exe and the screensaver file)

**Comment by tobyweston** [comment]
So the `config.yml` file will be found under the `user.home` folder. I'll add something in the log to show the full path.

In the meantime, from the stack trace above, it looks like you're not setting an environment variable for `TEAMCITY_HOME`. That's the minimum setup it needs. Once you've set one to point to your teamcity install, it'll scan the projects and create the config.

**Comment by Andrea** [comment]
> So the config.yml file will be found under the user.home folder. I'll add something in the log to 
> show the full path.
well, not there :D

> In the meantime, from the stack trace above, it looks like you're not setting an environment 
> variable for TEAMCITY_HOST. 

Editing the help now so this is more obvious on the readme file
When I do add the host and port I get this error
My environment settings are localhost for the host and port is 80 
```
Failed to create Yml configuration file (caused by an UnexpectedResponse Unexpected HTTP response from http://localhost/:80/guestAuth/app/rest/projects (404, Not Found)), falling back to use environment variables
Unexpected HTTP response from http://localhost/:80/guestAuth/app/rest/projects (404, Not Found) when monitoring monitoring unknown as a single aggregate

```


>Set it to the hostname of the teamcity server (ie localhost), no 
> need for http://localhost. Set the port using TEAMCITY_PORT if it's not the default of 8111. That's  
> the minimum setup it needs. 
Done this but it still doesn't work, 
> Once you've set one to point to your teamcity install, it'll scan the  
> projects and create the config.

ps, try radiate-1.0-20140618 (v15) until then...

this is the one that fails with "A Java exception has occurred " so nope :(

**Comment by tobyweston** [comment]
Thanks for the pull request.

It looks like your URL `http://localhost/:80/guestAuth/app/rest/projects` might be funny. Have you included a colon in the config? To override the port, just use the value `80`, and not `80`. Also, it probably doesn't work if you include the port in the `TEAMCITY_HOST` variable, it needs to go in the `TEAMCITY_PORT` one. 

hmmm... I just did a quick test and you'd get a different error than you're reporting if you'd misconfigured it like that... did you put a trailing slash after `localhost` in `TEAMCITY_HOST`? Is your TeamCity setup with [guest authorisation](https://confluence.jetbrains.com/display/TCD8/Guest+User) for the REST API? What version of TeamCity is it? 

What it will do if it can connect to your TeamCity is generate the `config.yml` and record what it's discovered; the project names etc. It will also list in the log where the file was created if you press `F1`.

**Comment by tobyweston** [comment]
A quick google, and I found a couple of public TeamCity instances. You could test it out but pointing to one of these. ie,

TEAMCITY_HOST=teamcity.bryntum.com
TEAMCITY_PORT=80

This is how I set them from my IDE, not there's no protocol or anything. 

![screen shot 2015-05-05 at 18 32 01](https://cloud.githubusercontent.com/assets/49573/7478816/364d4828-f355-11e4-8119-7e6393716a5c.png)

**Comment by tobyweston** [comment]
Did you manage to get things running? I'll close the issue if so.

**Comment by vorou** [comment]
Running these two, no luck:

```
radiate-1.0-20140618.062538-15.exe
radiate-1.0-20140618.062538-15.jar
```

Only empty `radiate.log` is created in `~\.radiate`. I'm on Windows 10 x64, Java 1.8.0_51.

**Comment by tobyweston** [comment]
hmm, sorry to hear that. I'm preparing a new version with some other tweaks and better error reporting. I suggest we have a go with that when it's ready.

**Comment by tobyweston** [comment]
Fixed in v2.0
